nx-dev/ui-customers/src/lib/customer-testimonial-carousel.tsx

```diff
@@ -0,0 +1,400 @@
+        {/* Left side - Quote or Metrics */}
+        <div className="col-span-2 hidden lg:block">
+          <div className="flex h-full flex-col justify-center space-y-8">
+            {currentTestimonial.metrics?.map((metric, index) => (
+              <div key={index} className="space-y-2">
+                <div className="border-l-2 border-sky-900/70 pl-4 text-3xl font-bold text-slate-700 dark:border-blue-300/60 dark:text-slate-200">
+                  {metric.value}
+                </div>
+                <div className="text-balance pl-[18px] text-lg text-slate-500 dark:text-slate-400">
+                  {metric.label}
+                </div>
+              </div>
+            ))}
+          </div>
+        </div>
+
+        {/* Right side - Video Card */}
+        <div className="col-span-2 md:col-span-3">
+          <div className="flex items-center gap-4">
+            {/* Prev Button Mobile only */}
+            <button
+              disabled={currentIndex === 0}
+              title={`See ${testimonials[currentIndex - 1]?.company} again!`}
+              className="flex h-12 w-12 items-center justify-center rounded-full p-2 transition hover:text-slate-950 disabled:pointer-events-none disabled:opacity-0 md:hidden dark:hover:text-white"
+              onClick={() => {
+                setCurrentIndex(
+                  (currentIndex - 1 + testimonials.length) % testimonials.length
+                );
+              }}
+            >
+              <ChevronLeftIcon className="h-8 w-8" />
+            </button>
+            <div
+              className="group relative h-full max-h-[450px] w-full cursor-pointer self-stretch overflow-hidden rounded-lg xl:shadow-2xl"
+              onClick={() => setIsOpen(true)}
+            >
+              {/* Thumbnail */}
+              <img
+                src={currentTestimonial.thumbnail}
+                alt={currentTestimonial.title}
+                className="aspect-video h-full w-full object-cover transition duration-300 group-hover:scale-105"
+              />
+
+              {/* Gradient Overlay */}
+              <div className="absolute inset-0 bg-gradient-to-t from-black/80 from-25% via-black/20" />
+
+              {/* Content Overlay */}
+              <div className="absolute bottom-0 left-0 p-4 md:p-8">
+                <h3 className="text-base font-semibold text-white md:text-2xl lg:text-3xl">
+                  {currentTestimonial.subtitle}
+                </h3>
+                <button className="mt-2 inline-flex items-center gap-1 rounded-full bg-white/10 px-4 py-2 text-sm font-medium text-white backdrop-blur-sm transition hover:bg-white/20 md:mt-4 md:gap-2">
+                  <PlayIcon className="size-3 md:size-5" />
+                  Watch video
+                </button>
+              </div>
+            </div>
+            {/* Next Button - Mobile only */}
+            <button
```

Comment:
Add a text to the button describing the action set with `sr-only` for accessibility purpose.

Reply:
done